vector/src/main/java/im/vector/activity/VectorRoomInviteMembersActivity.java

```diff
@@ -425,6 +426,24 @@ public boolean onPrepareOptionsMenu(Menu menu) {
         return super.onPrepareOptionsMenu(menu);
     }
 
+    @Override
+    public void onBackPressed() {
+        switch (mActionMode) {
+            case START_DIRECT_CHAT:
+            case SEND_INVITE:
+                Intent intent =  new Intent(VectorRoomInviteMembersActivity.this, VectorHomeActivity.class);
+                startActivity(intent);
+                finish();
+                break;
+            case RETURN_SELECTED_USER_IDS:
+                Intent intentWithResult = new Intent(VectorRoomInviteMembersActivity.this, TchapRoomCreationActivity.class);
```

Comment:
You could not force TchapRoomCreationActivity here
Your code should be similar to 
````
// Return the current list of the selected users in order to selected them if the picker is opened again
                    Intent intent = new Intent();
                    intent.putExtra(EXTRA_OUT_SELECTED_USER_IDS, mUserIdsToInvite);
                    setResult(RESULT_CANCELED, intent);
````